core/src/main/java/com/yubico/yubikit/core/smartcard/SmartCardProtocol.java

```diff
@@ -224,6 +224,11 @@ public byte[] sendAndReceive(Apdu command) throws IOException, ApduException {
   }
 
   public @Nullable DataEncryptor initScp(ScpKeyParams keyParams)
+      throws BadResponseException, ApduException, IOException {
+    return initScp(keyParams, false);
+  }
+
+  public @Nullable DataEncryptor initScp(ScpKeyParams keyParams, boolean shortApdus)
```

Comment:
Since we have `SmartCardConnection.isExtendedLengthApduSupported`, should we even expose `shortApdus` here? Would it be enough to just always use Extended APDUs if supported?